redis-consumer/redis-consumer.py

```diff
@@ -2,7 +2,6 @@
 
 import sys
 import os
-sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../listenstore"))
 sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
```

Comment:
While we're moving things around, lets move the redis_pubsub.py from the root into the listenstore as well. Then we can get rid of the second path appending bits.